modules/platform/nuxeo-fsexporter/src/test/java/org/nuxeo/io/fsexporter/test/TestFSExporterCollection.java

```diff
@@ -0,0 +1,102 @@
+package org.nuxeo.io.fsexporter.test;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.Serializable;
+
+import javax.inject.Inject;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.common.Environment;
+import org.nuxeo.ecm.collections.api.CollectionManager;
+import org.nuxeo.ecm.collections.core.test.CollectionFeature;
+import org.nuxeo.ecm.core.api.Blob;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.api.PathRef;
+import org.nuxeo.ecm.core.api.impl.blob.StringBlob;
+import org.nuxeo.io.fsexporter.FSExporter;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+
+@RunWith(FeaturesRunner.class)
+@Features(CollectionFeature.class)
+@Deploy("nuxeo-fsexporter")
+public class TestFSExporterCollection {
+
+    @Inject
+    CoreSession session;
```

Comment:
Could you make your fields `protected`?
Note: we're used to only use `protected` and `public` visibilities.